### ISSUE TITLE
Octoprint is scaling the rPi camera video stream

### ISSUE THREAD

**Issue by sundansx** [foosel/OctoPrint#2083]
**What were you doing?**

viewing piCam in browser
**What did you expect to happen?**
I expected the video window to be displayed at 640x480 as delivered by mjpeg streamer.  The slight reduction in scale make the image look less clear/sharp.

**various jpgs
![capture](https://user-images.githubusercontent.com/25075847/29449941-944960d0-83b1-11e7-9f0a-5acdf251d527.JPG)

** showing profile and printer settings.
What happened instead?

Video is being scaled to 588px × 441px.  cura video monitor shows the video in the correct resolution, albeit, upside down.
![capture](https://user-images.githubusercontent.com/25075847/29449654-69e6f416-83b0-11e7-9864-24878d655471.JPG)

Version: 1.3.4 (master branch)

**Printer model & used firmware incl. version**
Qidi Tech 1, sailfish 7.8 (new printer)

**Browser and Version of Browser, Operating System running Browser**
Firefox 54.0.1 (64 bit), Win10
Same thing happens with Chrome Version 47.0.2526.106

**Link to octoprint.log**

included in attached zip file
**Link to contents of terminal tab or serial.log**

not printing yet..
**Link to contents of Javascript console in the browser**
![capture](https://user-images.githubusercontent.com/25075847/29483052-244543ba-8453-11e7-9e08-e66af1a3276e.JPG)

NA?
**Screenshot(s) showing the problem:**
**Including screenshots and other files from above**
I have read the FAQ.

**Comment by ntoff** [comment]
a 640px wide picture won't fit in the table size octoprint uses for the tab content and OctoPrint doesn't do any sort of dynamic content resizing (you could try the screen squish plugin). It's only the image you see in the web interface that gets resized, the timelapse image should be clear and at their normal resolution.

**Comment by foosel** [comment]
As @ntoff already pointed out, there's no space for the full sized video feed in OctoPrint's control tab, so it scales to fit and there's no way around that (and now, just increasing the space there to allow the feed to get displayed at native resolution is no solution - people *are* in fact running full HD feeds...). So unless we are misunderstanding your problem here, this is working completely as intended.

**Comment by sundansx** [comment]
Ok, I see what is going on.  Thanks for the explanation.  A feature I did not know about that I discovered during this investigation was the double click functionality.  That looks like it gives me a full rez image (scaled to a larger size, which should work fine).  I will close this issue.  Thanks for your help Gina and ntoff.  Gina - ease back in, don't want to see you get burned out again.  Glad you had a nice vacation.
